src/main/java/org/folio/rest/impl/PostOrderLineHelper.java

```diff
@@ -210,6 +217,7 @@
             .thenAccept(id -> {
               if (id != null) {
                 reportingObject.setId(id);
+                reportingObject.setPoLineId(compPOL.getId());
```

Comment:
see my comments in the related acq-models pull request... reporting codes shouldn't have a po_line_id

Reply:
fixed according to comments